scripts/pre_deploy.sh

```diff
@@ -32,7 +32,7 @@ build_path="${__proj_dir}/build"
 _info "build_path: ${build_path}"
 _debug "$(find "${build_path}")"
 
-plugin_name="${__proj_dir##*/}"
+plugin_name=snaptel
```

Comment:
why is this hardcoded here as opposed to before?

Reply:
because line 35= snap-cli (plugin name), but binary is named 'snaptel`, so in release there is binary "snaptel_linux_x86_64", not "snap-cli_linux_x86_64" - and that cause that build failing.